interface/wx/textctrl.h

```diff
@@ -1659,8 +1659,8 @@ class wxTextCtrl : public wxControl,
         Returns @c true if text controls support reading and writing RTF (Rich
         Text Format).
 
-        Currently this function only returns true in wxOSX, which is the only
-        port implementing RTF support in wxTextCtrl.
+        This function only returns @true in wxOSX and wxMSW (if wxUSE_RICHEDIT is defined),
```

Comment:
We probably need to make this function non-static or at least add a non-static overload, that would check that the control uses `wxTE_RICH[2]`, as otherwise RTF-related functions still don't work.

Reply:
I changed it to non-static.